style.css

```diff
@@ -1,7 +1,125 @@
-html {
-  font-size: 10px;
-}
-title {
-  background-color: #3E3C3C;
-  color: blue;
+@media only screen and (max-width: 1500px) {
+  .wrapper {
+    margin: 0 auto;
+    background-color: #1c1a19;
+    background-image: url(./images/Illustration.png);
+    background-repeat: no-repeat;
+    background-position: right bottom;
+    display: flex;
+    flex-direction: column;
+    width: 375px;
+    height: 600px;
+  }
+
+  .container {
+    background-color: #3e3c3c;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 1em;
+  }
+
+  .hour {
+    font-family: 'Roboto', Verdana, Geneva, Tahoma, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    letter-spacing: -0.28px;
+    color: white;
+    flex: 1;
+  }
+
+  .indicators {
+    flex: 1;
+    align-self: center;
+    text-align: center;
+  }
+
+  .nav {
+    background-color: #3e3c3c;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .name {
+    font-family: 'Roboto', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    letter-spacing: -0.28px;
+    color: white;
+    flex: 1;
+  }
+
+  .hamburger {
+    flex: 1;
+    align-self: center;
+    text-align: center;
+  }
+
+  .info {
+    width: 327px;
+    height: 124px;
+    font-family: 'Crete Round', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 56px;
+    line-height: 64px;
+    letter-spacing: 0.37px;
+    color: #ff6b00;
+    padding-left: 24px;
+  }
+
+  .intro {
+    width: 327px;
+    height: 27px;
+    font-family: 'Roboto', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
+    display: flex;
+    align-items: center;
+    color: #fff5e1;
+    padding-left: 24px;
+  }
+
+  p {
+    width: 265px;
+    height: 123px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    color: white;
+    padding-left: 24px;
+    padding-bottom: 12px;
+  }
```

Comment:
- I think you did a great job trying to use the same font as provided in the Figma file. However, is not loading. Please check [this link](https://www.digitalocean.com/community/tutorials/how-to-load-and-use-custom-fonts-with-css) to get more information about how to add new fonts to your projects. :nerd_face:  :nerd_face: